internal/output/table.go

```diff
@@ -62,50 +62,104 @@ func ToTable(out Root, opts Options) ([]byte, error) {
 	return []byte(s), nil
 }
 
-func tableForBreakdown(breakdown Breakdown) string {
+func tableForBreakdown(breakdown Breakdown, fields []string) string {
 	t := table.NewWriter()
 	t.Style().Options.DrawBorder = false
 	t.Style().Options.SeparateColumns = false
 	t.Style().Options.SeparateRows = false
 	t.Style().Options.SeparateHeader = false
 	t.Style().Format.Header = text.FormatDefault
 
-	t.SetColumnConfigs([]table.ColumnConfig{
-		{Number: 1, Align: text.AlignLeft, AlignHeader: text.AlignLeft},
-		{Number: 2, Align: text.AlignRight, AlignHeader: text.AlignRight},
-		{Number: 3, Align: text.AlignLeft, AlignHeader: text.AlignLeft},
-		{Number: 4, Align: text.AlignRight, AlignHeader: text.AlignRight},
-	})
-
-	t.AppendHeader(table.Row{
+	var columns []table.ColumnConfig
+	var headers table.Row
+	headers = append(headers,
 		ui.UnderlineString("Name"),
-		ui.UnderlineString("Quantity"),
-		ui.UnderlineString("Unit"),
-		ui.UnderlineString("Monthly Cost"),
+	)
+
+	i := 1
+
+	columns = append(columns, table.ColumnConfig{
+		Number:      i,
+		Align:       text.AlignLeft,
+		AlignHeader: text.AlignLeft,
 	})
+	i++
+
+	if contains(fields, "price") {
+		headers = append(headers, ui.UnderlineString("Price"))
+		columns = append(columns, table.ColumnConfig{
+			Number:      i,
+			Align:       text.AlignLeft,
+			AlignHeader: text.AlignLeft,
```

Comment:
```suggestion
			Align:       text.AlignRight,
			AlignHeader: text.AlignRight,
```
Since this is a number it should be right-aligned